Localization: big docs update (#8961)
DevExpress/devextreme-documentation: concepts/05 UI Components/BarGauge/10 Visual Elements/40 Tooltips.md

Reviewer: ```suggestion
A tooltip is a small pop-up rectangle that displays information on a hovered bar in BarGauge. By default, the tooltip shows the value indicated by the hovered element. However, it is possible to display other information in the tooltip. You can customize the text of the tooltip and its appearance to suit your requirements.
```

You can change default tooltip settings by defining the properties within the **tooltip** configuration object. Some of these properties are categorized and listed below.

* **Text Customization and Formatting**    

> By default, a tooltip displays the value of a hovered UI component element without formatting. Set the [format](/api-reference/10%20UI%20Components/BaseWidget/1%20Configuration/tooltip/format.md '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/#format') property to format this value. When text customization is required, assign a function specifying the text to be shown to the [customizeTooltip](/api-reference/10%20UI%20Components/dxBarGauge/1%20Configuration/tooltip/customizeTooltip.md '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/#customizeTooltip') property. Learn more in the [Value Formatting](/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting '/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting/00%20Value%20Formatting.md') topic.

Reviewer: ```suggestion
By default, a tooltip displays the value of a hovered UI component element without formatting. Set the [format](/api-reference/10%20UI%20Components/BaseWidget/1%20Configuration/tooltip/format.md '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/#format') property to format this value. When text customization is required, assign a function that specifies the text to be shown to the [customizeTooltip](/api-reference/10%20UI%20Components/dxBarGauge/1%20Configuration/tooltip/customizeTooltip.md '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/#customizeTooltip') property. Learn more in the [Value Formatting](/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting '/concepts/Common/Localization%20and%20Globalization/10%20Value%20Formatting/00%20Value%20Formatting.md') topic.
```

* **Font Settings**    

> To customize the appearance of the tooltip text, define the font properties within the [font](/api-reference/10%20UI%20Components/BaseWidget/1%20Configuration/tooltip/font '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/font/') configuration object.

Reviewer: ```suggestion
To customize tooltip text appearance, define font properties within the [font](/api-reference/10%20UI%20Components/BaseWidget/1%20Configuration/tooltip/font '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/tooltip/font/') configuration object.
```
